packages/e2e/cypress/e2e/downloadCsv.cy.ts

```diff
@@ -130,7 +130,6 @@ describe('Download CSV on Explore', () => {
         cy.findByText('Loading chart').should('not.exist');
 
         cy.findByText('Export CSV').click();
-        cy.get('.bp4-popover2').contains('button', 'Export CSV').click();
```

Comment:
The e2e test is still failing on the next line. I suggest we increase the timeout to 10 seconds. 🤔